src/xenia/base/exception_handler_win.cc

```diff
@@ -40,9 +40,9 @@ LONG CALLBACK ExceptionHandlerCallback(PEXCEPTION_POINTERS ex_info) {
   thread_context.rip = ex_info->ContextRecord->Rip;
   thread_context.eflags = ex_info->ContextRecord->EFlags;
   std::memcpy(thread_context.int_registers, &ex_info->ContextRecord->Rax,
-              sizeof(thread_context.int_registers));
+              sizeof(*thread_context.int_registers));
```

Comment:
I think you can shut up PVS and also make it more clear by copying from  &ex_info->ContextRecord->int_registers instead of Rax?